data/brands/shop/bakery.json

```diff
@@ -1042,6 +1042,19 @@
         "shop": "bakery"
       }
     },
+    {
+      "displayName": "Kuhn",
+      "id": "kuhn-6950bd",
+      "locationSet": {"include": ["ch"]},
+      "tags": {
+        "bakehouse": "no",
+        "brand": "Kuhn",
+        "brand:wikidata": "Q111728345",
+        "cafe": "yes",
+        "drink:espresso": "served;takeaway",
```

Comment:
This is the only “drink:*” tag in this file, is every branch shop of Kuhn have espresso?

And, I don't think serve sepresso in a bakery is strange, many of them can serve this, and milk? coffee? We don't need to list all menu.

See https://github.com/osmlab/name-suggestion-index/pull/5888

Reply:
Removed it.